
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Documentation for the centrally provided Research Computing HPC services at UCL.">
      
      
        <meta name="author" content="UCL RCAS Team">
      
      
        <link rel="canonical" href="https://www.rc.ucl.ac.uk/docs/Wiki_Export/software/done/R_packages_available_on_Legion/">
      
      <link rel="icon" href="../../../../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.3.1, mkdocs-material-8.4.2">
    
    
      
        <title>R packages available on Legion - UCL Research Computing Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../../../assets/stylesheets/main.69437709.min.css">
      
        
        <link rel="stylesheet" href="../../../../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      
    
    
    
      
    
    
      <link rel="stylesheet" href="../../../../stylesheets/tweaks.css">
    
    <script>__md_scope=new URL("../../../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#r-and-bioconductor-packages-installed-on-rc-systems" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../../.." title="UCL Research Computing Documentation" class="md-header__button md-logo" aria-label="UCL Research Computing Documentation" data-md-component="logo">
      
  <img src="../../../../img/portico.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            UCL Research Computing Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              R packages available on Legion
            
          </span>
        </div>
      </div>
    </div>
    
      <form class="md-header__option" data-md-component="palette">
        
          
          
          <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
          
            <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5c-.84 0-1.65.15-2.39.42L12 2M3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29L3.34 7m.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14L3.36 17M20.65 7l-1.77 3.79a7.023 7.023 0 0 0-2.38-4.15l4.15.36m-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29L20.64 17M12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44L12 22Z"/></svg>
            </label>
          
        
          
          
          <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_2">
          
            <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3 3.19.09m3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95 2.06.05m-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31Z"/></svg>
            </label>
          
        
      </form>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/UCL-RITS/mkdocs-rc-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.1.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    UCL-RITS/mkdocs-rc-docs
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../../.." title="UCL Research Computing Documentation" class="md-nav__button md-logo" aria-label="UCL Research Computing Documentation" data-md-component="logo">
      
  <img src="../../../../img/portico.svg" alt="logo">

    </a>
    UCL Research Computing Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/UCL-RITS/mkdocs-rc-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.1.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    UCL-RITS/mkdocs-rc-docs
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Account_Services/" class="md-nav__link">
        Account Services
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Additional_Resource_Requests/" class="md-nav__link">
        Additional Resource Requests
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Contact_Us/" class="md-nav__link">
        Contact and Support
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Example_Jobscripts/" class="md-nav__link">
        Example Jobscripts
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Experienced_Users/" class="md-nav__link">
        Quick Start Guide for Experienced HPC Users
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Interactive_Jobs/" class="md-nav__link">
        Interactive Job Sessions
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Job_Results/" class="md-nav__link">
        Where do my results go?
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../New_Users/" class="md-nav__link">
        Guide for New Users
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Planned_Outages/" class="md-nav__link">
        Planned Outages
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Remote_Access/" class="md-nav__link">
        Remote Access
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Status_page/" class="md-nav__link">
        Status of machines
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../Terms_and_Conditions/" class="md-nav__link">
        Terms and Conditions
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../UCL_Service_For_Me/" class="md-nav__link">
        UCL Service For Me
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../../howto/" class="md-nav__link">
        How To
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_16" type="checkbox" id="__nav_16" >
      
      
      
      
        <label class="md-nav__link" for="__nav_16">
          Background
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Background" data-md-level="1">
        <label class="md-nav__title" for="__nav_16">
          <span class="md-nav__icon md-icon"></span>
          Background
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Background/Cluster_Computing/" class="md-nav__link">
        Cluster Computing
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_17" type="checkbox" id="__nav_17" >
      
      
      
      
        <label class="md-nav__link" for="__nav_17">
          Clusters
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Clusters" data-md-level="1">
        <label class="md-nav__title" for="__nav_17">
          <span class="md-nav__icon md-icon"></span>
          Clusters
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Myriad/" class="md-nav__link">
        Myriad
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Kathleen/" class="md-nav__link">
        Kathleen
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Thomas/" class="md-nav__link">
        MMM Thomas
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Michael/" class="md-nav__link">
        MMM Michael
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Acknowledging_RC_Systems/" class="md-nav__link">
        Acknowledging the Use of RC Systems
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Clusters/Young/" class="md-nav__link">
        MMM Hub Young
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_18" type="checkbox" id="__nav_18" >
      
      
      
      
        <label class="md-nav__link" for="__nav_18">
          Installed Software Lists
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Installed Software Lists" data-md-level="1">
        <label class="md-nav__title" for="__nav_18">
          <span class="md-nav__icon md-icon"></span>
          Installed Software Lists
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Installed_Software_Lists/module-packages/" class="md-nav__link">
        General Software Lists
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Installed_Software_Lists/python-packages/" class="md-nav__link">
        Python Packages
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Installed_Software_Lists/r-packages/" class="md-nav__link">
        R Packages
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_19" type="checkbox" id="__nav_19" >
      
      
      
      
        <label class="md-nav__link" for="__nav_19">
          Other Services
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Other Services" data-md-level="1">
        <label class="md-nav__title" for="__nav_19">
          <span class="md-nav__icon md-icon"></span>
          Other Services
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Other_Services/Aristotle/" class="md-nav__link">
        Aristotle
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Other_Services/RStudio/" class="md-nav__link">
        RStudio
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Other_Services/UCL_UK_e-Science_Certificates/" class="md-nav__link">
        UCL UK e-Science Certificates
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_20" type="checkbox" id="__nav_20" >
      
      
      
      
        <label class="md-nav__link" for="__nav_20">
          Paid-For Resources
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Paid-For Resources" data-md-level="1">
        <label class="md-nav__title" for="__nav_20">
          <span class="md-nav__icon md-icon"></span>
          Paid-For Resources
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Paid-For_Resources/How_to_Use/" class="md-nav__link">
        How to Use
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Paid-For_Resources/Purchasing_in_Myriad/" class="md-nav__link">
        Purchasing in Myriad
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_21" type="checkbox" id="__nav_21" >
      
      
      
      
        <label class="md-nav__link" for="__nav_21">
          Software Guides
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Software Guides" data-md-level="1">
        <label class="md-nav__title" for="__nav_21">
          <span class="md-nav__icon md-icon"></span>
          Software Guides
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/ANSYS/" class="md-nav__link">
        ANSYS
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/Matlab/" class="md-nav__link">
        MATLAB
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/Singularity/" class="md-nav__link">
        Singularity
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/R/" class="md-nav__link">
        R
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/Other_Software/" class="md-nav__link">
        Other Software
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Software_Guides/Installing_Software/" class="md-nav__link">
        Installing Software
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_22" type="checkbox" id="__nav_22" >
      
      
      
      
        <label class="md-nav__link" for="__nav_22">
          Walkthroughs
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Walkthroughs" data-md-level="1">
        <label class="md-nav__title" for="__nav_22">
          <span class="md-nav__icon md-icon"></span>
          Walkthroughs
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../../../Walkthroughs/Logging_In/" class="md-nav__link">
        Logging In
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
  <a href="https://github.com/UCL-RITS/mkdocs-rc-docs/blob/master/mkdocs-project-dir/docs/Wiki_Export/software/done/R_packages_available_on_Legion.md" title="Edit this page" class="md-content__button md-icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>



<h1 id="r-and-bioconductor-packages-installed-on-rc-systems">R and Bioconductor Packages installed on RC Systems<a class="headerlink" href="#r-and-bioconductor-packages-installed-on-rc-systems" title="Permanent link">§</a></h1>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Please consider replacing this with instructions on how to retrieve this list, instead of the list itself.</p>
</div>
<p>This is a list of the R add-on packages available with R 3.4.0 and
Bioconductor 3.5 on Legion, Grace, Thomas and Aristotle.</p>
<table>
<thead>
<tr>
<th align="center">Package</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">abc</td>
<td align="left">Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td align="center">abc.data</td>
<td align="left">Data Only: Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td align="center">abind</td>
<td align="left">Combine Multidimensional Arrays</td>
</tr>
<tr>
<td align="center">acepack</td>
<td align="left">ACE and AVAS for Selecting Multiple Regression Transformations</td>
</tr>
<tr>
<td align="center">adapt</td>
<td align="left">adapt -- multidimensional numerical integration</td>
</tr>
<tr>
<td align="center">ade4</td>
<td align="left">Analysis of Ecological Data : Exploratory and Euclidean Methods in Environmental Sciences</td>
</tr>
<tr>
<td align="center">adegenet</td>
<td align="left">Exploratory Analysis of Genetic and Genomic Data</td>
</tr>
<tr>
<td align="center">ADGofTest</td>
<td align="left">Anderson-Darling GoF test</td>
</tr>
<tr>
<td align="center">affxparser</td>
<td align="left">Affymetrix File Parsing SDK</td>
</tr>
<tr>
<td align="center">affy</td>
<td align="left">Methods for Affymetrix Oligonucleotide Arrays</td>
</tr>
<tr>
<td align="center">affydata</td>
<td align="left">Affymetrix Data for Demonstration Purpose</td>
</tr>
<tr>
<td align="center">affyio</td>
<td align="left">Tools for parsing Affymetrix data files</td>
</tr>
<tr>
<td align="center">affylmGUI</td>
<td align="left">GUI for limma package with Affymetrix microarrays</td>
</tr>
<tr>
<td align="center">affyPLM</td>
<td align="left">Methods for fitting probe-level models</td>
</tr>
<tr>
<td align="center">affyQCReport</td>
<td align="left">QC Report Generation for affyBatch objects</td>
</tr>
<tr>
<td align="center">akima</td>
<td align="left">Interpolation of Irregularly and Regularly Spaced Data</td>
</tr>
<tr>
<td align="center">annaffy</td>
<td align="left">Annotation tools for Affymetrix biological metadata</td>
</tr>
<tr>
<td align="center">annmap</td>
<td align="left">Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>
</tr>
<tr>
<td align="center">annotate</td>
<td align="left">Annotation for microarrays</td>
</tr>
<tr>
<td align="center">AnnotationDbi</td>
<td align="left">Annotation Database Interface</td>
</tr>
<tr>
<td align="center">AnnotationForge</td>
<td align="left">Code for Building Annotation Database Packages</td>
</tr>
<tr>
<td align="center">AnnotationHub</td>
<td align="left">Client to access AnnotationHub resources</td>
</tr>
<tr>
<td align="center">ape</td>
<td align="left">Analyses of Phylogenetics and Evolution</td>
</tr>
<tr>
<td align="center">aroma.affymetrix</td>
<td align="left">Analysis of Large Affymetrix Microarray Data Sets</td>
</tr>
<tr>
<td align="center">aroma.apd</td>
<td align="left">A Probe-Level Data File Format Used by 'aroma.affymetrix' [deprecated]</td>
</tr>
<tr>
<td align="center">aroma.core</td>
<td align="left">Core Methods and Classes Used by 'aroma.*' Packages Part of the Aroma Framework</td>
</tr>
<tr>
<td align="center">assertthat</td>
<td align="left">Easy pre and post assertions.</td>
</tr>
<tr>
<td align="center">backports</td>
<td align="left">Reimplementations of Functions Introduced Since R-3.0.0</td>
</tr>
<tr>
<td align="center">base</td>
<td align="left">The R Base Package</td>
</tr>
<tr>
<td align="center">base64</td>
<td align="left">Base64 Encoder and Decoder</td>
</tr>
<tr>
<td align="center">base64enc</td>
<td align="left">Tools for base64 encoding</td>
</tr>
<tr>
<td align="center">BaSTA</td>
<td align="left">Age-Specific Survival Analysis from Incomplete Capture-Recapture/Recovery Data</td>
</tr>
<tr>
<td align="center">BatchJobs</td>
<td align="left">Batch Computing with R</td>
</tr>
<tr>
<td align="center">bayesplot</td>
<td align="left">Plotting for Bayesian Models</td>
</tr>
<tr>
<td align="center">BBmisc</td>
<td align="left">Miscellaneous Helper Functions for B. Bischl</td>
</tr>
<tr>
<td align="center">beadarray</td>
<td align="left">Quality assessment and low-level analysis for Illumina BeadArray data</td>
</tr>
<tr>
<td align="center">beadarrayExampleData</td>
<td align="left">Example data for the beadarray package</td>
</tr>
<tr>
<td align="center">BeadDataPackR</td>
<td align="left">Compression of Illumina BeadArray data</td>
</tr>
<tr>
<td align="center">beanplot</td>
<td align="left">Visualization via Beanplots (like Boxplot/Stripchart/Violin Plot)</td>
</tr>
<tr>
<td align="center">BH</td>
<td align="left">Boost C++ Header Files</td>
</tr>
<tr>
<td align="center">BiasedUrn</td>
<td align="left">Biased Urn Model Distributions</td>
</tr>
<tr>
<td align="center">Biobase</td>
<td align="left">Biobase: Base functions for Bioconductor</td>
</tr>
<tr>
<td align="center">BiocGenerics</td>
<td align="left">S4 generic functions for Bioconductor</td>
</tr>
<tr>
<td align="center">BiocInstaller</td>
<td align="left">Install/Update Bioconductor, CRAN, and github Packages</td>
</tr>
<tr>
<td align="center">BiocParallel</td>
<td align="left">Bioconductor facilities for parallel evaluation</td>
</tr>
<tr>
<td align="center">biomaRt</td>
<td align="left">Interface to BioMart databases (e.g. Ensembl, COSMIC, Wormbase and Gramene)</td>
</tr>
<tr>
<td align="center">Biostrings</td>
<td align="left">String objects representing biological sequences, and matching algorithms</td>
</tr>
<tr>
<td align="center">biovizBase</td>
<td align="left">Basic graphic utilities for visualization of genomic data.</td>
</tr>
<tr>
<td align="center">bitops</td>
<td align="left">Bitwise Operations</td>
</tr>
<tr>
<td align="center">blockmodeling</td>
<td align="left">An R package for Generalized and classical blockmodeling of valued networks</td>
</tr>
<tr>
<td align="center">boot</td>
<td align="left">Bootstrap Functions (Originally by Angelo Canty for S)</td>
</tr>
<tr>
<td align="center">brew</td>
<td align="left">Templating Framework for Report Generation</td>
</tr>
<tr>
<td align="center">BSgenome</td>
<td align="left">Infrastructure for Biostrings-based genome data packages and support for efficient SNP representation</td>
</tr>
<tr>
<td align="center">BSgenome.Hsapiens.UCSC.hg19</td>
<td align="left">Full genome sequences for Homo sapiens (UCSC version hg19)</td>
</tr>
<tr>
<td align="center">bsseq</td>
<td align="left">Analyze, manage and store bisulfite sequencing data</td>
</tr>
<tr>
<td align="center">bumphunter</td>
<td align="left">Bump Hunter</td>
</tr>
<tr>
<td align="center">car</td>
<td align="left">Companion to Applied Regression</td>
</tr>
<tr>
<td align="center">caret</td>
<td align="left">Classification and Regression Training</td>
</tr>
<tr>
<td align="center">Category</td>
<td align="left">Category Analysis</td>
</tr>
<tr>
<td align="center">caTools</td>
<td align="left">Tools: moving window statistics, GIF, Base64, ROC AUC, etc.</td>
</tr>
<tr>
<td align="center">ChAMP</td>
<td align="left">Chip Analysis Methylation Pipeline for Illumina HumanMethylation450 and EPIC</td>
</tr>
<tr>
<td align="center">ChAMPdata</td>
<td align="left">Data Packages for ChAMP package</td>
</tr>
<tr>
<td align="center">checkmate</td>
<td align="left">Fast and Versatile Argument Checks</td>
</tr>
<tr>
<td align="center">class</td>
<td align="left">Functions for Classification</td>
</tr>
<tr>
<td align="center">cluster</td>
<td align="left">"Finding Groups in Data": Cluster Analysis Extended Rousseeuw et al.</td>
</tr>
<tr>
<td align="center">cmprsk</td>
<td align="left">Subdistribution Analysis of Competing Risks</td>
</tr>
<tr>
<td align="center">coda</td>
<td align="left">Output Analysis and Diagnostics for MCMC</td>
</tr>
<tr>
<td align="center">codetools</td>
<td align="left">Code Analysis Tools for R</td>
</tr>
<tr>
<td align="center">colorRamps</td>
<td align="left">Builds color tables</td>
</tr>
<tr>
<td align="center">colorspace</td>
<td align="left">Color Space Manipulation</td>
</tr>
<tr>
<td align="center">colourpicker</td>
<td align="left">A Colour Picker Widget for Shiny Apps, RStudio, R-markdown, and 'htmlwidgets'</td>
</tr>
<tr>
<td align="center">combinat</td>
<td align="left">combinatorics utilities</td>
</tr>
<tr>
<td align="center">compiler</td>
<td align="left">The R Compiler Package</td>
</tr>
<tr>
<td align="center">copula</td>
<td align="left">Multivariate Dependence with Copulas</td>
</tr>
<tr>
<td align="center">copynumber</td>
<td align="left">Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>
</tr>
<tr>
<td align="center">corpcor</td>
<td align="left">Efficient Estimation of Covariance and (Partial) Correlation</td>
</tr>
<tr>
<td align="center">corrplot</td>
<td align="left">Visualization of a Correlation Matrix</td>
</tr>
<tr>
<td align="center">crayon</td>
<td align="left">Colored Terminal Output</td>
</tr>
<tr>
<td align="center">curl</td>
<td align="left">A Modern and Flexible Web Client for R</td>
</tr>
<tr>
<td align="center">data.table</td>
<td align="left">Extension of Data.frame</td>
</tr>
<tr>
<td align="center">datasets</td>
<td align="left">The R Datasets Package</td>
</tr>
<tr>
<td align="center">DBI</td>
<td align="left">R Database Interface</td>
</tr>
<tr>
<td align="center">deldir</td>
<td align="left">Delaunay Triangulation and Dirichlet (Voronoi) Tessellation</td>
</tr>
<tr>
<td align="center">dendextend</td>
<td align="left">Extending R's Dendrogram Functionality</td>
</tr>
<tr>
<td align="center">DEoptimR</td>
<td align="left">Differential Evolution Optimization in Pure R</td>
</tr>
<tr>
<td align="center">DESeq</td>
<td align="left">Differential gene expression analysis based on the negative binomial distribution</td>
</tr>
<tr>
<td align="center">DESeq2</td>
<td align="left">Differential gene expression analysis based on the negative binomial distribution</td>
</tr>
<tr>
<td align="center">Design</td>
<td align="left">Design Package</td>
</tr>
<tr>
<td align="center">DEXSeq</td>
<td align="left">Inference of differential exon usage in RNA-Seq</td>
</tr>
<tr>
<td align="center">dichromat</td>
<td align="left">Color Schemes for Dichromats</td>
</tr>
<tr>
<td align="center">digest</td>
<td align="left">Create Compact Hash Digests of R Objects</td>
</tr>
<tr>
<td align="center">diptest</td>
<td align="left">Hartigan's Dip Test Statistic for Unimodality - Corrected</td>
</tr>
<tr>
<td align="center">DMRcate</td>
<td align="left">Methylation array and sequencing spatial analysis methods</td>
</tr>
<tr>
<td align="center">DMRcatedata</td>
<td align="left">Data Package for DMRcate package</td>
</tr>
<tr>
<td align="center">DNAcopy</td>
<td align="left">DNA copy number data analysis</td>
</tr>
<tr>
<td align="center">doMC</td>
<td align="left">Foreach Parallel Adaptor for 'parallel'</td>
</tr>
<tr>
<td align="center">doParallel</td>
<td align="left">Foreach Parallel Adaptor for the 'parallel' Package</td>
</tr>
<tr>
<td align="center">doRNG</td>
<td align="left">Generic Reproducible Parallel Backend for foreach Loops</td>
</tr>
<tr>
<td align="center">dplyr</td>
<td align="left">A Grammar of Data Manipulation</td>
</tr>
<tr>
<td align="center">DSS</td>
<td align="left">Dispersion shrinakge for sequencing data.</td>
</tr>
<tr>
<td align="center">DT</td>
<td align="left">A Wrapper of the JavaScript Library 'DataTables'</td>
</tr>
<tr>
<td align="center">dygraphs</td>
<td align="left">Interface to 'Dygraphs' Interactive Time Series Charting Library</td>
</tr>
<tr>
<td align="center">dynamicTreeCut</td>
<td align="left">Methods for Detection of Clusters in Hierarchical Clustering Dendrograms</td>
</tr>
<tr>
<td align="center">DynDoc</td>
<td align="left">Dynamic document tools</td>
</tr>
<tr>
<td align="center">e1071</td>
<td align="left">Misc Functions of the Department of Statistics, Probability Theory Group (Formerly: E1071), TU Wien</td>
</tr>
<tr>
<td align="center">easyRNASeq</td>
<td align="left">Count summarization and normalization for RNA-Seq data</td>
</tr>
<tr>
<td align="center">EBSeq</td>
<td align="left">An R package for gene and isoform differential expression analysis of RNA-seq data</td>
</tr>
<tr>
<td align="center">edgeR</td>
<td align="left">Empirical Analysis of Digital Gene Expression Data in R</td>
</tr>
<tr>
<td align="center">effects</td>
<td align="left">Effect Displays for Linear, Generalized Linear, and Other Models</td>
</tr>
<tr>
<td align="center">ellipse</td>
<td align="left">Functions for drawing ellipses and ellipse-like confidence regions</td>
</tr>
<tr>
<td align="center">ensembldb</td>
<td align="left">Utilities to create and use an Ensembl based annotation database</td>
</tr>
<tr>
<td align="center">Epi</td>
<td align="left">A Package for Statistical Analysis in Epidemiology</td>
</tr>
<tr>
<td align="center">etm</td>
<td align="left">Empirical Transition Matrix</td>
</tr>
<tr>
<td align="center">evaluate</td>
<td align="left">Parsing and Evaluation Tools that Provide More Details than the Default</td>
</tr>
<tr>
<td align="center">expm</td>
<td align="left">Matrix Exponential</td>
</tr>
<tr>
<td align="center">FactoMineR</td>
<td align="left">Multivariate Exploratory Data Analysis and Data Mining</td>
</tr>
<tr>
<td align="center">fail</td>
<td align="left">File Abstraction Interface Layer (FAIL)</td>
</tr>
<tr>
<td align="center">fastcluster</td>
<td align="left">Fast Hierarchical Clustering Routines for R and Python</td>
</tr>
<tr>
<td align="center">fastICA</td>
<td align="left">FastICA Algorithms to perform ICA and Projection Pursuit</td>
</tr>
<tr>
<td align="center">fastmatch</td>
<td align="left">Fast match() function</td>
</tr>
<tr>
<td align="center">FDb.InfiniumMethylation.hg19</td>
<td align="left">Annotation package for Illumina Infinium DNA methylation probes</td>
</tr>
<tr>
<td align="center">fdrtool</td>
<td align="left">Estimation of (Local) False Discovery Rates and Higher Criticism</td>
</tr>
<tr>
<td align="center">FEM</td>
<td align="left">Identification of Functional Epigenetic Modules</td>
</tr>
<tr>
<td align="center">fields</td>
<td align="left">Tools for Spatial Data</td>
</tr>
<tr>
<td align="center">flashClust</td>
<td align="left">Implementation of optimal hierarchical clustering</td>
</tr>
<tr>
<td align="center">flexmix</td>
<td align="left">Flexible Mixture Modeling</td>
</tr>
<tr>
<td align="center">foreach</td>
<td align="left">Provides Foreach Looping Construct for R</td>
</tr>
<tr>
<td align="center">foreign</td>
<td align="left">Read Data Stored by Minitab, S, SAS, SPSS, Stata, Systat, Weka, dBase, ...</td>
</tr>
<tr>
<td align="center">Formula</td>
<td align="left">Extended Model Formulas</td>
</tr>
<tr>
<td align="center">fpc</td>
<td align="left">Flexible Procedures for Clustering</td>
</tr>
<tr>
<td align="center">futile.logger</td>
<td align="left">A Logging Utility for R</td>
</tr>
<tr>
<td align="center">futile.options</td>
<td align="left">Futile options management</td>
</tr>
<tr>
<td align="center">future</td>
<td align="left">A Future API for R</td>
</tr>
<tr>
<td align="center">gam</td>
<td align="left">Generalized Additive Models</td>
</tr>
<tr>
<td align="center">gamlss</td>
<td align="left">Generalised Additive Models for Location Scale and Shape</td>
</tr>
<tr>
<td align="center">gamlss.data</td>
<td align="left">GAMLSS Data</td>
</tr>
<tr>
<td align="center">gamlss.dist</td>
<td align="left">Distributions to be Used for GAMLSS Modelling</td>
</tr>
<tr>
<td align="center">gamlss.mx</td>
<td align="left">Fitting Mixture Distributions with GAMLSS</td>
</tr>
<tr>
<td align="center">gamlss.nl</td>
<td align="left">Fitting non linear parametric GAMLSS models</td>
</tr>
<tr>
<td align="center">gcrma</td>
<td align="left">Background Adjustment Using Sequence Information</td>
</tr>
<tr>
<td align="center">gdata</td>
<td align="left">Various R Programming Tools for Data Manipulation</td>
</tr>
<tr>
<td align="center">GenABEL</td>
<td align="left">genome-wide SNP association analysis</td>
</tr>
<tr>
<td align="center">GenABEL.data</td>
<td align="left">Package contains data which is used by GenABEL example and test functions</td>
</tr>
<tr>
<td align="center">genefilter</td>
<td align="left">genefilter: methods for filtering genes from high-throughput experiments</td>
</tr>
<tr>
<td align="center">geneLenDataBase</td>
<td align="left">Lengths of mRNA transcripts for a number of genomes</td>
</tr>
<tr>
<td align="center">GeneNet</td>
<td align="left">Modeling and Inferring Gene Networks</td>
</tr>
<tr>
<td align="center">geneplotter</td>
<td align="left">Graphics related functions for Bioconductor</td>
</tr>
<tr>
<td align="center">genetics</td>
<td align="left">Population Genetics</td>
</tr>
<tr>
<td align="center">GenomeGraphs</td>
<td align="left">Plotting genomic information from Ensembl</td>
</tr>
<tr>
<td align="center">GenomeInfoDb</td>
<td align="left">Utilities for manipulating chromosome and other 'seqname' identifiers</td>
</tr>
<tr>
<td align="center">genomeIntervals</td>
<td align="left">Operations on genomic intervals</td>
</tr>
<tr>
<td align="center">GenomicAlignments</td>
<td align="left">Representation and manipulation of short genomic alignments</td>
</tr>
<tr>
<td align="center">GenomicFeatures</td>
<td align="left">Tools for making and manipulating transcript centric annotations</td>
</tr>
<tr>
<td align="center">GenomicRanges</td>
<td align="left">Representation and manipulation of genomic intervals and variables defined along a genome</td>
</tr>
<tr>
<td align="center">GEOquery</td>
<td align="left">Get data from NCBI Gene Expression Omnibus (GEO)</td>
</tr>
<tr>
<td align="center">GGally</td>
<td align="left">Extension to 'ggplot2'</td>
</tr>
<tr>
<td align="center">ggplot2</td>
<td align="left">Create Elegant Data Visualisations Using the Grammar of Graphics</td>
</tr>
<tr>
<td align="center">ggvis</td>
<td align="left">Interactive Grammar of Graphics</td>
</tr>
<tr>
<td align="center">glmnet</td>
<td align="left">Lasso and Elastic-Net Regularized Generalized Linear Models</td>
</tr>
<tr>
<td align="center">globals</td>
<td align="left">Identify Global Objects in R Expressions</td>
</tr>
<tr>
<td align="center">gmodels</td>
<td align="left">Various R Programming Tools for Model Fitting</td>
</tr>
<tr>
<td align="center">GO.db</td>
<td align="left">A set of annotation maps describing the entire Gene Ontology</td>
</tr>
<tr>
<td align="center">goseq</td>
<td align="left">Gene Ontology analyser for RNA-seq and other length biased data</td>
</tr>
<tr>
<td align="center">GOstats</td>
<td align="left">Tools for manipulating GO and microarrays.</td>
</tr>
<tr>
<td align="center">gplots</td>
<td align="left">Various R Programming Tools for Plotting Data</td>
</tr>
<tr>
<td align="center">graph</td>
<td align="left">graph: A package to handle graph data structures</td>
</tr>
<tr>
<td align="center">graphics</td>
<td align="left">The R Graphics Package</td>
</tr>
<tr>
<td align="center">grDevices</td>
<td align="left">The R Graphics Devices and Support for Colours and Fonts</td>
</tr>
<tr>
<td align="center">grid</td>
<td align="left">The Grid Graphics Package</td>
</tr>
<tr>
<td align="center">gridBase</td>
<td align="left">Integration of base and grid graphics</td>
</tr>
<tr>
<td align="center">gridExtra</td>
<td align="left">Miscellaneous Functions for &quot;Grid&quot; Graphics</td>
</tr>
<tr>
<td align="center">GSEABase</td>
<td align="left">Gene set enrichment data structures and methods</td>
</tr>
<tr>
<td align="center">gsl</td>
<td align="left">wrapper for the Gnu Scientific Library</td>
</tr>
<tr>
<td align="center">gsmoothr</td>
<td align="left">Smoothing tools</td>
</tr>
<tr>
<td align="center">gtable</td>
<td align="left">Arrange 'Grobs' in Tables</td>
</tr>
<tr>
<td align="center">gtools</td>
<td align="left">Various R Programming Tools</td>
</tr>
<tr>
<td align="center">Gviz</td>
<td align="left">Plotting data and annotation information along genomic coordinates</td>
</tr>
<tr>
<td align="center">HAC</td>
<td align="left">Estimation, Simulation and Visualization of Hierarchical Archimedean Copulae (HAC)</td>
</tr>
<tr>
<td align="center">haplo.stats</td>
<td align="left">Statistical Analysis of Haplotypes with Traits and Covariates when Linkage Phase is Ambiguous</td>
</tr>
<tr>
<td align="center">hexbin</td>
<td align="left">Hexagonal Binning Routines</td>
</tr>
<tr>
<td align="center">hgu95av2.db</td>
<td align="left">Affymetrix Human Genome U95 Set annotation data (chip hgu95av2)</td>
</tr>
<tr>
<td align="center">HI</td>
<td align="left">Simulation from distributions supported by nested hyperplanes</td>
</tr>
<tr>
<td align="center">highr</td>
<td align="left">Syntax Highlighting for R Source Code</td>
</tr>
<tr>
<td align="center">Hmisc</td>
<td align="left">Harrell Miscellaneous</td>
</tr>
<tr>
<td align="center">HotDeckImputation</td>
<td align="left">Hot Deck Imputation Methods for Missing Data</td>
</tr>
<tr>
<td align="center">htmlTable</td>
<td align="left">Advanced Tables for Markdown/HTML</td>
</tr>
<tr>
<td align="center">htmltools</td>
<td align="left">Tools for HTML</td>
</tr>
<tr>
<td align="center">htmlwidgets</td>
<td align="left">HTML Widgets for R</td>
</tr>
<tr>
<td align="center">httpuv</td>
<td align="left">HTTP and WebSocket Server Library</td>
</tr>
<tr>
<td align="center">httr</td>
<td align="left">Tools for Working with URLs and HTTP</td>
</tr>
<tr>
<td align="center">hugene10stprobeset.db</td>
<td align="left">Affymetrix hugene10 annotation data (chip hugene10stprobeset)</td>
</tr>
<tr>
<td align="center">hugene10sttranscriptcluster.db</td>
<td align="left">Affymetrix hugene10 annotation data (chip hugene10sttranscriptcluster)</td>
</tr>
<tr>
<td align="center">hwriter</td>
<td align="left">HTML Writer - Outputs R objects in HTML format</td>
</tr>
<tr>
<td align="center">igraph</td>
<td align="left">Network Analysis and Visualization</td>
</tr>
<tr>
<td align="center">Illumina450ProbeVariants.db</td>
<td align="left">Annotation Package combining variant data from 1000 Genomes Project for Illumina HumanMethylation450 Bead Chip probes</td>
</tr>
<tr>
<td align="center">IlluminaHumanMethylation450kanno.ilmn12.hg19</td>
<td align="left">Annotation for Illumina's 450k methylation arrays</td>
</tr>
<tr>
<td align="center">IlluminaHumanMethylation450kmanifest</td>
<td align="left">Annotation for Illumina's 450k methylation arrays</td>
</tr>
<tr>
<td align="center">IlluminaHumanMethylationEPICanno.ilm10b2.hg19</td>
<td align="left">Annotation for Illumina's EPIC methylation arrays</td>
</tr>
<tr>
<td align="center">IlluminaHumanMethylationEPICmanifest</td>
<td align="left">Manifest for Illumina's EPIC methylation arrays</td>
</tr>
<tr>
<td align="center">illuminaHumanv4.db</td>
<td align="left">Illumina HumanHT12v4 annotation data (chip illuminaHumanv4)</td>
</tr>
<tr>
<td align="center">illuminaio</td>
<td align="left">Parsing Illumina Microarray Output Files</td>
</tr>
<tr>
<td align="center">impute</td>
<td align="left">impute: Imputation for microarray data</td>
</tr>
<tr>
<td align="center">INLA</td>
<td align="left">Functions which Allow to Perform Full Bayesian Analysis of Latent Gaussian Models using Integrated Nested Laplace Approximations</td>
</tr>
<tr>
<td align="center">inline</td>
<td align="left">Functions to Inline C, C++, Fortran Function Calls from R</td>
</tr>
<tr>
<td align="center">interactiveDisplayBase</td>
<td align="left">Base package for enabling powerful shiny web displays of Bioconductor objects</td>
</tr>
<tr>
<td align="center">intervals</td>
<td align="left">Tools for Working with Points and Intervals</td>
</tr>
<tr>
<td align="center">IRanges</td>
<td align="left">Infrastructure for manipulating intervals on sequences</td>
</tr>
<tr>
<td align="center">irlba</td>
<td align="left">Fast Truncated SVD, PCA and Symmetric Eigendecomposition for Large Dense and Sparse Matrices</td>
</tr>
<tr>
<td align="center">isva</td>
<td align="left">Independent Surrogate Variable Analysis</td>
</tr>
<tr>
<td align="center">iterators</td>
<td align="left">Provides Iterator Construct for R</td>
</tr>
<tr>
<td align="center">jsonlite</td>
<td align="left">A Robust, High Performance JSON Parser and Generator for R</td>
</tr>
<tr>
<td align="center">KEGG.db</td>
<td align="left">A set of annotation maps for KEGG</td>
</tr>
<tr>
<td align="center">kernlab</td>
<td align="left">Kernel-Based Machine Learning Lab</td>
</tr>
<tr>
<td align="center">KernSmooth</td>
<td align="left">Functions for Kernel Smoothing Supporting Wand &amp; Jones (1995)</td>
</tr>
<tr>
<td align="center">knitr</td>
<td align="left">A General-Purpose Package for Dynamic Report Generation in R</td>
</tr>
<tr>
<td align="center">labeling</td>
<td align="left">Axis Labeling</td>
</tr>
<tr>
<td align="center">lambda.r</td>
<td align="left">Modeling Data with Functional Programming</td>
</tr>
<tr>
<td align="center">lattice</td>
<td align="left">Trellis Graphics for R</td>
</tr>
<tr>
<td align="center">latticeExtra</td>
<td align="left">Extra Graphical Utilities Based on Lattice</td>
</tr>
<tr>
<td align="center">lazyeval</td>
<td align="left">Lazy (Non-Standard) Evaluation</td>
</tr>
<tr>
<td align="center">leaps</td>
<td align="left">regression subset selection</td>
</tr>
<tr>
<td align="center">LearnBayes</td>
<td align="left">Functions for Learning Bayesian Inference</td>
</tr>
<tr>
<td align="center">limma</td>
<td align="left">Linear Models for Microarray Data</td>
</tr>
<tr>
<td align="center">listenv</td>
<td align="left">Environments Behaving (Almost) as Lists</td>
</tr>
<tr>
<td align="center">lme4</td>
<td align="left">Linear Mixed-Effects Models using 'Eigen' and S4</td>
</tr>
<tr>
<td align="center">locfit</td>
<td align="left">Local Regression, Likelihood and Density Estimation.</td>
</tr>
<tr>
<td align="center">longitudinal</td>
<td align="left">Analysis of Multiple Time Course Data</td>
</tr>
<tr>
<td align="center">loo</td>
<td align="left">Efficient Leave-One-Out Cross-Validation and WAIC for Bayesian Models</td>
</tr>
<tr>
<td align="center">lpSolve</td>
<td align="left">Interface to 'Lp_solve' v. 5.5 to Solve Linear/Integer Programs</td>
</tr>
<tr>
<td align="center">LSD</td>
<td align="left">Lots of Superior Depictions</td>
</tr>
<tr>
<td align="center">lumi</td>
<td align="left">BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>
</tr>
<tr>
<td align="center">made4</td>
<td align="left">Multivariate analysis of microarray data using ADE4</td>
</tr>
<tr>
<td align="center">magrittr</td>
<td align="left">A Forward-Pipe Operator for R</td>
</tr>
<tr>
<td align="center">maps</td>
<td align="left">Draw Geographical Maps</td>
</tr>
<tr>
<td align="center">maptree</td>
<td align="left">Mapping, pruning, and graphing tree models</td>
</tr>
<tr>
<td align="center">markdown</td>
<td align="left">'Markdown' Rendering for R</td>
</tr>
<tr>
<td align="center">marray</td>
<td align="left">Exploratory analysis for two-color spotted microarray data</td>
</tr>
<tr>
<td align="center">MASS</td>
<td align="left">Support Functions and Datasets for Venables and Ripley's MASS</td>
</tr>
<tr>
<td align="center">Matrix</td>
<td align="left">Sparse and Dense Matrix Classes and Methods</td>
</tr>
<tr>
<td align="center">MatrixModels</td>
<td align="left">Modelling with Sparse And Dense Matrices</td>
</tr>
<tr>
<td align="center">matrixStats</td>
<td align="left">Functions that Apply to Rows and Columns of Matrices (and to Vectors)</td>
</tr>
<tr>
<td align="center">mclust</td>
<td align="left">Gaussian Mixture Modelling for Model-Based Clustering, Classification, and Density Estimation</td>
</tr>
<tr>
<td align="center">mcmc</td>
<td align="left">Markov Chain Monte Carlo</td>
</tr>
<tr>
<td align="center">memoise</td>
<td align="left">Memoisation of Functions</td>
</tr>
<tr>
<td align="center">metafor</td>
<td align="left">Meta-Analysis Package for R</td>
</tr>
<tr>
<td align="center">methods</td>
<td align="left">Formal Methods and Classes</td>
</tr>
<tr>
<td align="center">methylumi</td>
<td align="left">Handle Illumina methylation data</td>
</tr>
<tr>
<td align="center">mgcv</td>
<td align="left">Mixed GAM Computation Vehicle with GCV/AIC/REML Smoothness Estimation</td>
</tr>
<tr>
<td align="center">mime</td>
<td align="left">Map Filenames to MIME Types</td>
</tr>
<tr>
<td align="center">minfi</td>
<td align="left">Analyze Illumina Infinium DNA methylation arrays</td>
</tr>
<tr>
<td align="center">miniUI</td>
<td align="left">Shiny UI Widgets for Small Screens</td>
</tr>
<tr>
<td align="center">minqa</td>
<td align="left">Derivative-free optimization algorithms by quadratic approximation</td>
</tr>
<tr>
<td align="center">missMethyl</td>
<td align="left">Analysing Illumina HumanMethylation BeadChip Data</td>
</tr>
<tr>
<td align="center">mitools</td>
<td align="left">Tools for multiple imputation of missing data</td>
</tr>
<tr>
<td align="center">mlr</td>
<td align="left">Machine Learning in R</td>
</tr>
<tr>
<td align="center">mnormt</td>
<td align="left">The Multivariate Normal and t Distributions</td>
</tr>
<tr>
<td align="center">ModelMetrics</td>
<td align="left">Rapid Calculation of Model Metrics</td>
</tr>
<tr>
<td align="center">modeltools</td>
<td align="left">Tools and Classes for Statistical Models</td>
</tr>
<tr>
<td align="center">msm</td>
<td align="left">Multi-State Markov and Hidden Markov Models in Continuous Time</td>
</tr>
<tr>
<td align="center">mstate</td>
<td align="left">Data Preparation, Estimation and Prediction in Multi-State Models</td>
</tr>
<tr>
<td align="center">multcomp</td>
<td align="left">Simultaneous Inference in General Parametric Models</td>
</tr>
<tr>
<td align="center">multtest</td>
<td align="left">Resampling-based multiple hypothesis testing</td>
</tr>
<tr>
<td align="center">munsell</td>
<td align="left">Utilities for Using Munsell Colours</td>
</tr>
<tr>
<td align="center">mvtnorm</td>
<td align="left">Multivariate Normal and t Distributions</td>
</tr>
<tr>
<td align="center">NetworkAnalysis</td>
<td align="left">Statistical inference on populations of weighted or unweighted networks.</td>
</tr>
<tr>
<td align="center">nleqslv</td>
<td align="left">Solve Systems of Nonlinear Equations</td>
</tr>
<tr>
<td align="center">nlme</td>
<td align="left">Linear and Nonlinear Mixed Effects Models</td>
</tr>
<tr>
<td align="center">nloptr</td>
<td align="left">R interface to NLopt</td>
</tr>
<tr>
<td align="center">NMF</td>
<td align="left">Algorithms and Framework for Nonnegative Matrix Factorization (NMF)</td>
</tr>
<tr>
<td align="center">nnet</td>
<td align="left">Feed-Forward Neural Networks and Multinomial Log-Linear Models</td>
</tr>
<tr>
<td align="center">nnls</td>
<td align="left">The Lawson-Hanson algorithm for non-negative least squares (NNLS)</td>
</tr>
<tr>
<td align="center">nor1mix</td>
<td align="left">Normal (1-d) Mixture Models (S3 Classes and Methods)</td>
</tr>
<tr>
<td align="center">numDeriv</td>
<td align="left">Accurate Numerical Derivatives</td>
</tr>
<tr>
<td align="center">nutshell</td>
<td align="left">Data for &quot;R in a Nutshell&quot;</td>
</tr>
<tr>
<td align="center">nutshell.audioscrobbler</td>
<td align="left">Audioscrobbler data for &quot;R in a Nutshell&quot;</td>
</tr>
<tr>
<td align="center">nutshell.bbdb</td>
<td align="left">Baseball Database for &quot;R in a Nutshell&quot;</td>
</tr>
<tr>
<td align="center">OPE</td>
<td align="left">Outer-product emulator</td>
</tr>
<tr>
<td align="center">openssl</td>
<td align="left">Toolkit for Encryption, Signatures and Certificates Based on OpenSSL</td>
</tr>
<tr>
<td align="center">org.Hs.eg.db</td>
<td align="left">Genome wide annotation for Human</td>
</tr>
<tr>
<td align="center">packrat</td>
<td align="left">A Dependency Management System for Projects and their R Package Dependencies</td>
</tr>
<tr>
<td align="center">parallel</td>
<td align="left">Support for Parallel computation in R</td>
</tr>
<tr>
<td align="center">parallelMap</td>
<td align="left">Unified Interface to Parallelization Back-Ends</td>
</tr>
<tr>
<td align="center">ParamHelpers</td>
<td align="left">Helpers for Parameters in Black-Box Optimization, Tuning and Machine Learning</td>
</tr>
<tr>
<td align="center">parcor</td>
<td align="left">Regularized estimation of partial correlation matrices</td>
</tr>
<tr>
<td align="center">pbkrtest</td>
<td align="left">Parametric Bootstrap and Kenward Roger Based Methods for Mixed Model Comparison</td>
</tr>
<tr>
<td align="center">pcaPP</td>
<td align="left">Robust PCA by Projection Pursuit</td>
</tr>
<tr>
<td align="center">pegas</td>
<td align="left">Population and Evolutionary Genetics Analysis System</td>
</tr>
<tr>
<td align="center">permute</td>
<td align="left">Functions for Generating Restricted Permutations of Data</td>
</tr>
<tr>
<td align="center">phangorn</td>
<td align="left">Phylogenetic Analysis in R</td>
</tr>
<tr>
<td align="center">pixmap</td>
<td align="left">Bitmap Images (``Pixel Maps'')</td>
</tr>
<tr>
<td align="center">pkgmaker</td>
<td align="left">Package development utilities</td>
</tr>
<tr>
<td align="center">PKI</td>
<td align="left">Public Key Infrastucture for R Based on the X.509 Standard</td>
</tr>
<tr>
<td align="center">plogr</td>
<td align="left">The 'plog' C++ Logging Library</td>
</tr>
<tr>
<td align="center">plotly</td>
<td align="left">Create Interactive Web Graphics via 'plotly.js'</td>
</tr>
<tr>
<td align="center">pls</td>
<td align="left">Partial Least Squares and Principal Component Regression</td>
</tr>
<tr>
<td align="center">plyr</td>
<td align="left">Tools for Splitting, Applying and Combining Data</td>
</tr>
<tr>
<td align="center">polspline</td>
<td align="left">Polynomial Spline Routines</td>
</tr>
<tr>
<td align="center">poweRlaw</td>
<td align="left">Analysis of Heavy Tailed Distributions</td>
</tr>
<tr>
<td align="center">ppls</td>
<td align="left">Penalized Partial Least Squares</td>
</tr>
<tr>
<td align="center">prabclus</td>
<td align="left">Functions for Clustering of Presence-Absence, Abundance and Multilocus Genetic Data</td>
</tr>
<tr>
<td align="center">pracma</td>
<td align="left">Practical Numerical Math Functions</td>
</tr>
<tr>
<td align="center">praise</td>
<td align="left">Praise Users</td>
</tr>
<tr>
<td align="center">preprocessCore</td>
<td align="left">A collection of pre-processing functions</td>
</tr>
<tr>
<td align="center">prettyunits</td>
<td align="left">Pretty, Human Readable Formatting of Quantities</td>
</tr>
<tr>
<td align="center">progress</td>
<td align="left">Terminal Progress Bars</td>
</tr>
<tr>
<td align="center">proto</td>
<td align="left">Prototype Object-Based Programming</td>
</tr>
<tr>
<td align="center">PSCBS</td>
<td align="left">Analysis of Parent-Specific DNA Copy Numbers</td>
</tr>
<tr>
<td align="center">pspline</td>
<td align="left">Penalized Smoothing Splines</td>
</tr>
<tr>
<td align="center">psych</td>
<td align="left">Procedures for Psychological, Psychometric, and Personality Research</td>
</tr>
<tr>
<td align="center">purrr</td>
<td align="left">Functional Programming Tools</td>
</tr>
<tr>
<td align="center">quadprog</td>
<td align="left">Functions to solve Quadratic Programming Problems.</td>
</tr>
<tr>
<td align="center">quantreg</td>
<td align="left">Quantile Regression</td>
</tr>
<tr>
<td align="center">qvalue</td>
<td align="left">Q-value estimation for false discovery rate control</td>
</tr>
<tr>
<td align="center">R.cache</td>
<td align="left">Fast and Light-Weight Caching (Memoization) of Objects and Results to Speed Up Computations</td>
</tr>
<tr>
<td align="center">R.devices</td>
<td align="left">Unified Handling of Graphics Devices</td>
</tr>
<tr>
<td align="center">R.filesets</td>
<td align="left">Easy Handling of and Access to Files Organized in Structured Directories</td>
</tr>
<tr>
<td align="center">R.huge</td>
<td align="left">Methods for Accessing Huge Amounts of Data [deprecated]</td>
</tr>
<tr>
<td align="center">R.methodsS3</td>
<td align="left">S3 Methods Simplified</td>
</tr>
<tr>
<td align="center">R.oo</td>
<td align="left">R Object-Oriented Programming with or without References</td>
</tr>
<tr>
<td align="center">R.rsp</td>
<td align="left">Dynamic Generation of Scientific Reports</td>
</tr>
<tr>
<td align="center">R.utils</td>
<td align="left">Various Programming Utilities</td>
</tr>
<tr>
<td align="center">R2HTML</td>
<td align="left">HTML Exportation for R Objects</td>
</tr>
<tr>
<td align="center">R2jags</td>
<td align="left">Using R to Run 'JAGS'</td>
</tr>
<tr>
<td align="center">R2WinBUGS</td>
<td align="left">Running 'WinBUGS' and 'OpenBUGS' from 'R' / 'S-PLUS'</td>
</tr>
<tr>
<td align="center">R6</td>
<td align="left">Classes with Reference Semantics</td>
</tr>
<tr>
<td align="center">randomForest</td>
<td align="left">Breiman and Cutler's Random Forests for Classification and Regression</td>
</tr>
<tr>
<td align="center">RBGL</td>
<td align="left">An interface to the BOOST graph library</td>
</tr>
<tr>
<td align="center">RColorBrewer</td>
<td align="left">ColorBrewer Palettes</td>
</tr>
<tr>
<td align="center">Rcpp</td>
<td align="left">Seamless R and C++ Integration</td>
</tr>
<tr>
<td align="center">RcppArmadillo</td>
<td align="left">'Rcpp' Integration for the 'Armadillo' Templated Linear Algebra Library</td>
</tr>
<tr>
<td align="center">RcppEigen</td>
<td align="left">'Rcpp' Integration for the 'Eigen' Templated Linear Algebra Library</td>
</tr>
<tr>
<td align="center">RcppGSL</td>
<td align="left">'Rcpp' Integration for 'GNU GSL' Vectors and Matrices</td>
</tr>
<tr>
<td align="center">RCurl</td>
<td align="left">General Network (HTTP/FTP/...) Client Interface for R</td>
</tr>
<tr>
<td align="center">RefFreeEWAS</td>
<td align="left">EWAS using Reference-Free DNA Methylation Mixture Deconvolution</td>
</tr>
<tr>
<td align="center">registry</td>
<td align="left">Infrastructure for R Package Registries</td>
</tr>
<tr>
<td align="center">relaimpo</td>
<td align="left">Relative importance of regressors in linear models</td>
</tr>
<tr>
<td align="center">Repitools</td>
<td align="left">Epigenomic tools</td>
</tr>
<tr>
<td align="center">reshape</td>
<td align="left">Flexibly Reshape Data</td>
</tr>
<tr>
<td align="center">reshape2</td>
<td align="left">Flexibly Reshape Data: A Reboot of the Reshape Package</td>
</tr>
<tr>
<td align="center">rgdal</td>
<td align="left">Bindings for the Geospatial Data Abstraction Library</td>
</tr>
<tr>
<td align="center">rgeos</td>
<td align="left">Interface to Geometry Engine - Open Source (GEOS)</td>
</tr>
<tr>
<td align="center">rgl</td>
<td align="left">3D Visualization Using OpenGL</td>
</tr>
<tr>
<td align="center">Rglpk</td>
<td align="left">R/GNU Linear Programming Kit Interface</td>
</tr>
<tr>
<td align="center">ridge</td>
<td align="left">Ridge Regression with automatic selection of the penalty parameter</td>
</tr>
<tr>
<td align="center">Ringo</td>
<td align="left">R Investigation of ChIP-chip Oligoarrays</td>
</tr>
<tr>
<td align="center">rjags</td>
<td align="left">Bayesian Graphical Models using MCMC</td>
</tr>
<tr>
<td align="center">RJSONIO</td>
<td align="left">Serialize R objects to JSON, JavaScript Object Notation</td>
</tr>
<tr>
<td align="center">rlecuyer</td>
<td align="left">R Interface to RNG with Multiple Streams</td>
</tr>
<tr>
<td align="center">Rmpi</td>
<td align="left">Interface (Wrapper) to MPI (Message-Passing Interface)</td>
</tr>
<tr>
<td align="center">rms</td>
<td align="left">Regression Modeling Strategies</td>
</tr>
<tr>
<td align="center">RMySQL</td>
<td align="left">Database Interface and 'MySQL' Driver for R</td>
</tr>
<tr>
<td align="center">RNetCDF</td>
<td align="left">Interface to NetCDF Datasets</td>
</tr>
<tr>
<td align="center">rngtools</td>
<td align="left">Utility functions for working with Random Number Generators</td>
</tr>
<tr>
<td align="center">robustbase</td>
<td align="left">Basic Robust Statistics</td>
</tr>
<tr>
<td align="center">ROC</td>
<td align="left">utilities for ROC, with uarray focus</td>
</tr>
<tr>
<td align="center">rpart</td>
<td align="left">Recursive Partitioning and Regression Trees</td>
</tr>
<tr>
<td align="center">RPMM</td>
<td align="left">Recursively Partitioned Mixture Model</td>
</tr>
<tr>
<td align="center">Rsamtools</td>
<td align="left">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
</tr>
<tr>
<td align="center">rsconnect</td>
<td align="left">Deployment Interface for R Markdown Documents and Shiny Applications</td>
</tr>
<tr>
<td align="center">Rsolnp</td>
<td align="left">General Non-Linear Optimization</td>
</tr>
<tr>
<td align="center">RSQLite</td>
<td align="left">'SQLite' Interface for R</td>
</tr>
<tr>
<td align="center">rstan</td>
<td align="left">R Interface to Stan</td>
</tr>
<tr>
<td align="center">rstanarm</td>
<td align="left">Bayesian Applied Regression Modeling via Stan</td>
</tr>
<tr>
<td align="center">rstantools</td>
<td align="left">Tools for Developing R Packages Interfacing with 'Stan'</td>
</tr>
<tr>
<td align="center">rstudioapi</td>
<td align="left">Safely Access the RStudio API</td>
</tr>
<tr>
<td align="center">Rsubread</td>
<td align="left">Subread sequence alignment for R</td>
</tr>
<tr>
<td align="center">rtracklayer</td>
<td align="left">R interface to genome browsers and their annotation tracks</td>
</tr>
<tr>
<td align="center">ruv</td>
<td align="left">Detect and Remove Unwanted Variation using Negative Controls</td>
</tr>
<tr>
<td align="center">S4Vectors</td>
<td align="left">S4 implementation of vectors and lists</td>
</tr>
<tr>
<td align="center">saemix</td>
<td align="left">Stochastic Approximation Expectation Maximization (SAEM) algorithm</td>
</tr>
<tr>
<td align="center">sampling</td>
<td align="left">Survey Sampling</td>
</tr>
<tr>
<td align="center">sandwich</td>
<td align="left">Robust Covariance Matrix Estimators</td>
</tr>
<tr>
<td align="center">scales</td>
<td align="left">Scale Functions for Visualization</td>
</tr>
<tr>
<td align="center">scatterplot3d</td>
<td align="left">3D Scatter Plot</td>
</tr>
<tr>
<td align="center">schoolmath</td>
<td align="left">Functions and datasets for math used in school</td>
</tr>
<tr>
<td align="center">segmented</td>
<td align="left">Regression Models with Breakpoints/Changepoints Estimation</td>
</tr>
<tr>
<td align="center">sendmailR</td>
<td align="left">send email using R</td>
</tr>
<tr>
<td align="center">seqinr</td>
<td align="left">Biological Sequences Retrieval and Analysis</td>
</tr>
<tr>
<td align="center">seqLogo</td>
<td align="left">Sequence logos for DNA sequence alignments</td>
</tr>
<tr>
<td align="center">shiny</td>
<td align="left">Web Application Framework for R</td>
</tr>
<tr>
<td align="center">shinyjs</td>
<td align="left">Easily Improve the User Experience of Your Shiny Apps in Seconds</td>
</tr>
<tr>
<td align="center">shinystan</td>
<td align="left">Interactive Visual and Numerical Diagnostics and Posterior Analysis for Bayesian Models</td>
</tr>
<tr>
<td align="center">shinythemes</td>
<td align="left">Themes for Shiny</td>
</tr>
<tr>
<td align="center">ShortRead</td>
<td align="left">FASTQ input and manipulation</td>
</tr>
<tr>
<td align="center">siggenes</td>
<td align="left">Multiple testing using SAM and Efron's empirical Bayes approaches</td>
</tr>
<tr>
<td align="center">simpleaffy</td>
<td align="left">Very simple high level analysis of Affymetrix data</td>
</tr>
<tr>
<td align="center">slam</td>
<td align="left">Sparse Lightweight Arrays and Matrices</td>
</tr>
<tr>
<td align="center">sn</td>
<td align="left">The Skew-Normal and Skew-t Distributions</td>
</tr>
<tr>
<td align="center">snow</td>
<td align="left">Simple Network of Workstations</td>
</tr>
<tr>
<td align="center">snowfall</td>
<td align="left">Easier cluster computing (based on snow).</td>
</tr>
<tr>
<td align="center">softImpute</td>
<td align="left">Matrix Completion via Iterative Soft-Thresholded SVD</td>
</tr>
<tr>
<td align="center">sourcetools</td>
<td align="left">Tools for Reading, Tokenizing and Parsing R Code</td>
</tr>
<tr>
<td align="center">sp</td>
<td align="left">Classes and Methods for Spatial Data</td>
</tr>
<tr>
<td align="center">spam</td>
<td align="left">SPArse Matrix</td>
</tr>
<tr>
<td align="center">SparseM</td>
<td align="left">Sparse Linear Algebra</td>
</tr>
<tr>
<td align="center">spatial</td>
<td align="left">Functions for Kriging and Point Pattern Analysis</td>
</tr>
<tr>
<td align="center">spdep</td>
<td align="left">Spatial Dependence: Weighting Schemes, Statistics and Models</td>
</tr>
<tr>
<td align="center">splines</td>
<td align="left">Regression Spline Functions and Classes</td>
</tr>
<tr>
<td align="center">stabledist</td>
<td align="left">Stable Distribution Functions</td>
</tr>
<tr>
<td align="center">StanHeaders</td>
<td align="left">C++ Header Files for Stan</td>
</tr>
<tr>
<td align="center">statmod</td>
<td align="left">Statistical Modeling</td>
</tr>
<tr>
<td align="center">stats</td>
<td align="left">The R Stats Package</td>
</tr>
<tr>
<td align="center">stats4</td>
<td align="left">Statistical Functions using S4 Classes</td>
</tr>
<tr>
<td align="center">stringdist</td>
<td align="left">Approximate String Matching and String Distance Functions</td>
</tr>
<tr>
<td align="center">stringi</td>
<td align="left">Character String Processing Facilities</td>
</tr>
<tr>
<td align="center">stringr</td>
<td align="left">Simple, Consistent Wrappers for Common String Operations</td>
</tr>
<tr>
<td align="center">SummarizedExperiment</td>
<td align="left">SummarizedExperiment container</td>
</tr>
<tr>
<td align="center">survey</td>
<td align="left">Analysis of Complex Survey Samples</td>
</tr>
<tr>
<td align="center">survival</td>
<td align="left">Survival Analysis</td>
</tr>
<tr>
<td align="center">sva</td>
<td align="left">Surrogate Variable Analysis</td>
</tr>
<tr>
<td align="center">tcltk</td>
<td align="left">Tcl/Tk Interface</td>
</tr>
<tr>
<td align="center">testthat</td>
<td align="left">Unit Testing for R</td>
</tr>
<tr>
<td align="center">tgp</td>
<td align="left">Bayesian Treed Gaussian Process Models</td>
</tr>
<tr>
<td align="center">TH.data</td>
<td align="left">TH's Data Archive</td>
</tr>
<tr>
<td align="center">threejs</td>
<td align="left">Interactive 3D Scatter Plots, Networks and Globes</td>
</tr>
<tr>
<td align="center">tibble</td>
<td align="left">Simple Data Frames</td>
</tr>
<tr>
<td align="center">tidyr</td>
<td align="left">Easily Tidy Data with <code>spread()</code> and <code>gather()</code> Functions</td>
</tr>
<tr>
<td align="center">tkrplot</td>
<td align="left">TK Rplot</td>
</tr>
<tr>
<td align="center">tools</td>
<td align="left">Tools for Package Development</td>
</tr>
<tr>
<td align="center">trimcluster</td>
<td align="left">Cluster analysis with trimming</td>
</tr>
<tr>
<td align="center">truncnorm</td>
<td align="left">Truncated normal distribution</td>
</tr>
<tr>
<td align="center">tseries</td>
<td align="left">Time Series Analysis and Computational Finance</td>
</tr>
<tr>
<td align="center">TxDb.Hsapiens.UCSC.hg19.knownGene</td>
<td align="left">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td align="center">utils</td>
<td align="left">The R Utils Package</td>
</tr>
<tr>
<td align="center">VariantAnnotation</td>
<td align="left">Annotation of Genetic Variants</td>
</tr>
<tr>
<td align="center">vegan</td>
<td align="left">Community Ecology Package</td>
</tr>
<tr>
<td align="center">VGAM</td>
<td align="left">Vector Generalized Linear and Additive Models</td>
</tr>
<tr>
<td align="center">viridis</td>
<td align="left">Default Color Maps from 'matplotlib'</td>
</tr>
<tr>
<td align="center">viridisLite</td>
<td align="left">Default Color Maps from 'matplotlib' (Lite Version)</td>
</tr>
<tr>
<td align="center">vsn</td>
<td align="left">Variance stabilization and calibration for microarray data</td>
</tr>
<tr>
<td align="center">wateRmelon</td>
<td align="left">Illumina 450 methylation array normalization and metrics</td>
</tr>
<tr>
<td align="center">WGCNA</td>
<td align="left">Weighted Correlation Network Analysis</td>
</tr>
<tr>
<td align="center">whisker</td>
<td align="left">for R, logicless templating</td>
</tr>
<tr>
<td align="center">widgetTools</td>
<td align="left">Creates an interactive tcltk widget</td>
</tr>
<tr>
<td align="center">XML</td>
<td align="left">Tools for Parsing and Generating XML Within R and S-Plus</td>
</tr>
<tr>
<td align="center">xps</td>
<td align="left">Processing and Analysis of Affymetrix Oligonucleotide Arrays including Exon Arrays, Whole Genome Arrays and Plate Arrays</td>
</tr>
<tr>
<td align="center">xtable</td>
<td align="left">Export Tables to LaTeX or HTML</td>
</tr>
<tr>
<td align="center">xts</td>
<td align="left">eXtensible Time Series</td>
</tr>
<tr>
<td align="center">XVector</td>
<td align="left">Representation and manpulation of external sequences</td>
</tr>
<tr>
<td align="center">yaml</td>
<td align="left">Methods to Convert R Data to YAML and Back</td>
</tr>
<tr>
<td align="center">zlibbioc</td>
<td align="left">An R packaged zlib-1.2.5</td>
</tr>
<tr>
<td align="center">zoo</td>
<td align="left">S3 Infrastructure for Regular and Irregular Time Series (Z's Ordered Observations)</td>
</tr>
</tbody>
</table>




              
            </article>
            
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
        
        <a href="../Quantum_Espresso/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Quantum Espresso" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Quantum Espresso
            </div>
          </div>
        </a>
      
      
        
        <a href="../../unedited/Running_CP2K_on_Legion/" class="md-footer__link md-footer__link--next" aria-label="Next: Running CP2K on Legion" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              Running CP2K on Legion
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2014 - 2020 UCL Research Computing Group
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
      
      
    
    <a href="https://github.com/UCL-RITS" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.1.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../../../..", "features": ["instant"], "search": "../../../../assets/javascripts/workers/search.ecf98df9.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../../../../assets/javascripts/bundle.9c69f0bc.min.js"></script>
      
    
  </body>
</html>